setup.py

```diff
@@ -71,6 +71,11 @@ def get_version():
             "pyvisa-py==0.5.3",
             "qm-qua>=1.0.1",
             "qualang-tools>=0.13.1",
+            "zhinst-toolkit==0.5.0",
+            "laboneq==2.1.0",  # 2.2.0
+            "laboneq-token==1.0.0",  #
+            # from laboneq.simple import install_token
+            # install_token("LabOneQ.AllQuantumInOne.")
```

Comment:
It's already weird to have here something that is not just a string (and of course the first line would not work, since a statement can't be part of a list), but if you're not using it, drop it.

Reply:
I was not sure how to say for the people that wanted to use it that for it to work you need to run that with the token I wrote there

Reply:
Add it to the driver Sphinx page :) It is as public as in the code...

Actually, why do we need a token?

Reply:
Ok, then we might want to purge it from the code...

Level 1, drop it from the branch, most people won't scan the repo for secrets.
If really needed, there are also ways to rewrite the history to purge completely from the Git database.

Reply:
However, you could do even better:
```py
import laboneq._token

laboneq._token.is_valid_token = lambda _token: True
```

Would actually work as a token itself...

Can you write them an email? Otherwise, I'm going to open an issue on their repo, but they might prefer to keep it private...

Reply:
I don't think the token is that important it just gets sended automatically when you introduce the email, and its the same as I checked with stavros.

Reply:
Ok, but then why did they write 100 lines of code, to store the token automatically, or check it from an env var, if it can be completely bypassed by everyone?

My guess is that they were gating the access to count people using the code, and keep track of them, by having them filling the form as soon as they need to run something.

However, if you believe not to be relevant, you can just drop the token from the present status of `zhinst` branch (who cares if it's still in commits), and put the two lines above in the driver, such that we won't need a token any longer :)

Reply:
Actually, the full solution is:
```
import os
import laboneq._token

os.environ["LABONEQ_TOKEN"] = "ciao come va?" # or other random text
laboneq._token.is_valid_token = lambda _token: True
```

Without setting the env var you would need the same to install some token (again, whatever non-empty text), otherwise the function `get_token()` would raise an error when called by `token_check()`.

Reply:
Yeah I think is what you said, because as far as I remenber stavros was running things on their simulation mode and I only needed the token when I started connecting to the devices. Yeah, I don't think the token is an issue with them so I will add that.